core/src/main/java/org/apache/accumulo/core/clientImpl/OfflineIterator.java

```diff
@@ -300,13 +295,9 @@ private TabletMetadata getTabletFiles(Range nextRange) {
     SamplerConfigurationImpl samplerConfImpl =
         SamplerConfigurationImpl.newSamplerConfig(acuTableConf);
 
-    if (scannerSamplerConfigImpl != null
-        && ((samplerConfImpl != null && !scannerSamplerConfigImpl.equals(samplerConfImpl))
-            || samplerConfImpl == null)) {
+    if (scannerSamplerConfigImpl != null && (!scannerSamplerConfigImpl.equals(samplerConfImpl))) {
```

Comment:
Can remove these extra parens:

```suggestion
    if (scannerSamplerConfigImpl != null && !scannerSamplerConfigImpl.equals(samplerConfImpl)) {
```